setup.py

```diff
@@ -3,10 +3,10 @@
 dependencies = [
     "multidict==5.1.0",  # Avoid 5.2.0 due to Avast
     "aiofiles==0.7.0",  # Async IO for files
-    "blspy==1.0.8",  # Signature library
-    "chiavdf==1.0.3",  # timelord and vdf verification
-    "chiabip158==1.0",  # bip158-style wallet filters
-    "chiapos==1.0.7",  # proof of space
+    "blspy==1.0.9",  # Signature library
+    "chiavdf==1.0.4",  # timelord and vdf verification
+    "chiabip158==1.1",  # bip158-style wallet filters
+    "chiapos==1.0.8",  # proof of space
```

Comment:
bumping these dependencies seem unrelated to supporting python 3.10. are there other python-3.10 fixes in those dependencies?

Reply:
I think these are updates for these to provide wheels.  I'm not sure if there are other changes (nor entirely sure about that).  They are being updated in https://github.com/Chia-Network/chia-blockchain/pull/9808 separately anyways and presumably that goes in first.